minimesos/src/main/java/com/containersol/minimesos/container/ContainerName.java

```diff
@@ -48,6 +48,8 @@ public static boolean hasRoleInCluster(String[] dockerNames, String clusterId, S
     }
 
     /**
+     * @param containerName TODO(mwl)
```

Comment:
![INFO](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-info.png) Complete the task associated to this TODO comment. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://ci.infra.container-solutions.com/sonar/coding_rules#rule_key=squid%3AS1135)